README.md

```diff
@@ -107,15 +115,16 @@ packages:
   - package: dbt-labs/dbt_utils
     version: [">=0.8.0", "<0.9.0"]
 ```
-# 🙌 Contributions and Maintenance
-## Package Maintenance
-The Fivetran team maintaining this package **only** maintains the latest version of the package. We highly recommend you stay consistent with the [latest version](https://hub.getdbt.com/fivetran/github_source/latest/) of the package and refer to the [CHANGELOG](/CHANGELOG.md) and release notes for more information on changes across versions.
 
-## Contributions
+## 🙌 Contributions and Maintenance
```

Comment:
See my note above RE: Level 2 headers being questions. I'm stuck on a good question for this one. What about "What else should I know?"